Assets/Data/Localization/German/MenuLocalization.json

```
@@ -126,19 +126,19 @@
         },
         {
             "id": "settingsMenu.controls.topView",
-            "text": "Flugzeug-Ansicht"
+            "text": "Draufsicht"
         },
         {
             "id": "settingsMenu.controls.extraControls",
-            "text": "Extra Steuerungen"
+            "text": "Weitere Steuerungen"
         },
         {
             "id": "settingsMenu.controls.makeDayTime",
-            "text": "Setze zu Tag"
+            "text": "Mach es Tag"
```

```suggestion
            "text": "Tageszeit"
```
Sounds a bit nicer

---

Maybe that's not clear enough. I thought about that too.

---

In my first translation try I had
```suggestion
            "text": "Zu Tag wechseln"
```

In my opinion it sounds better than "Mach es Tag" but its still not perfect

---

I don't really know which one sounds better.

---

"Zu Tageszeit wechseln" / "Zu Nachtzeit wechseln"?

Or " zu" → "auf" possibly. But there's usually a better word than "machen". :)

---

> "Zu Tageszeit wechseln" / "Zu Nachtzeit wechseln"?
> 
> Or " zu" → "auf" possibly. But there's usually a better word than "machen". :)

In my opinion the best solution. I changed it.